docs/install.md

```diff
@@ -12,6 +12,7 @@
 - [macOS](#macos)
 - [Docker](#docker)
 - [Helm](#helm)
+- [Windows](#windows)
```

Comment:
@BrunoQuaresma The preview for this PR seems to show the old build rather than the new one for some reason, maybe a missing env var in the build settings or something?

![image](https://user-images.githubusercontent.com/52710/130375783-30a4a641-4eea-4735-92b7-0354118659cc.png)

Reply:
Fix: https://github.com/cdr/m/pull/9646